partials/time_to_retirement.html

```diff
@@ -15,111 +15,121 @@
             </div>
         </div>
         <div class="row cannot-retire col-xs-12 col-sm-8 col-sm-offset-2" ng-show="retirement.never_retire_shown">
+        	<div class="big-number-bg">
+                <div class="big-number-wrapper">
+                	<i class="fa fa-remove icon-answer"></i>

```

Comment:
Yeah, it's for Font Awesome: http://fortawesome.github.io/Font-Awesome/examples/